### PR TITLE
python3Packages.pyxnat: init at 1.3

### PR DESCRIPTION
**Pull request by bcdarwin**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by 9999years — APPROVED** [review]
LGTM.

**Review by jonringer — COMMENTED** [review]
please rebase, conflicts

**Comment by bcdarwin** [comment]
Rebased onto master.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5906,6 +5906,8 @@ in {
     });
   in if isPy3k then pyxattr' else pyxattr_2;
 
+  pyxnat = callPackage ../development/python-modules/pyxnat { };
+
   pyx = callPackage ../development/python-modules/pyx { };
 
   pyxdg = callPackage ../development/python-modules/pyxdg { };
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix:5913]: this should be lower, at least after `pyxdg`

Comment (bcdarwin) [reply pkgs/top-level/python-packages.nix:5913]: Is it OK if I move `pyx` slightly?

Comment (bcdarwin) [reply pkgs/top-level/python-packages.nix]: Never mind, moved.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 89963 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python37Packages.pyxnat</li>    <li>python38Packages.pyxnat</li>  </ul></details>

**Pull request merged**